rootfs/defaults/jmr.conf

```
@@ -0,0 +1,10 @@
+{{ $JMR_DIRECTORY := .Env.JMR_DIRECTORY | default "/data" -}}
+
+jitsi-multitrack-recorder {
+  recording {
+    {{ if .Env.JMR_FORMAT -}}
```

```suggestion
    {{- if .Env.JMR_FORMAT }}
```
pretty sure you want to chomp whitespace backwards here instead of forwards, to leave the whitespace in the next line